### PR TITLE
Remove no longer needed Win32 configure flags

### PR DESCRIPTION
**Pull request by sergeyklay**
- Remove no longer needed Win32 configure flags
- Fixed indents for `appveyor.yml`
- Setting up AppVeyor `on_failure` target
- Create release package via PowerShell

### CODE REVIEW

File: config.w32 [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // vim:ft=javascript
 
-ARG_ENABLE("psr", "enable psr support", "no");
+ARG_ENABLE("psr", "Enable psr support", "no");
+ARG_ENABLE("sanitizer", "Enable address sanitizer extension", "no")
```
Comment (jbboehr) [inline config.w32]: @sergeyklay I asked in windev on IRC a few months ago, and they said this issue was supposed to be fixed soon, so it may work now.

Comment (sergeyklay) [reply config.w32]: @jbboehr Should I just remove `--enable-sanitizer` from the `config.w32 ` ?

Comment (jbboehr) [reply config.w32]: Assuming windev actually fixed it, we should be able to remove both of `config-profile` and `sanitizer`.

Comment (sergeyklay) [reply config.w32]: Let me do it :sunglasses:

**Comment by sergeyklay** [comment]
@jbboehr Done

**Comment by sergeyklay** [comment]
@jbboehr Just moved creating release package to the PowerShell module

**Pull request closed without merging**